projects/plugins/wpcomsh/wpcomsh.php

```diff
@@ -27,6 +27,8 @@
 
 require_once __DIR__ . '/plugin-hotfixes.php';
 
+require_once __DIR__ . '/woa.php';
```

Comment:
If the contents of this are only used in a WP_CLI context, better to include them only in that context (L136/L138).